src/rgw/rgw_common.h

```diff
@@ -1537,6 +1537,10 @@ struct rgw_obj_key {
     instance = i;
   }
 
+  string get_instance() const {
+    return instance;
+  }
```

Comment:
can return as `const string&` - but `instance` is public, so this function might be overkill